### PR TITLE
Add unit test for AlterInstanceHandler to cover the thrown exception

### LINKED ISSUE #16359: [OpenForce 2022] Add more unit test for `AlterInstanceHandler`
Hi community,

This issue is for https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441

We need to add more unit tests for AlterInstanceHandler to cover the thrown exception

- [ ] AlterInstanceHandler

Example FYI
- AlterTrafficRuleHandlerTest

### PR DESCRIPTION
**Pull request by ismailalammar**
Fixes #16359.

Changes proposed in this pull request:
- adding unit test to check alterInstanceHandler with not supported key.
- adding unit test to check alterInstanceHandler with no persistence configuration found.
- adding unit test to check alterInstanceHandler with Not exist InstanceId

### CODE REVIEW

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package org.apache.shardingsphere.proxy.backend.text.distsql.ral.common.updatable;
```
Comment (strongduanmu) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java:1]: @ismailalammar Please add apache license header.

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.apache.shardingsphere.mode.persist.PersistRepository;
+import org.apache.shardingsphere.proxy.backend.context.ProxyContext;
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.*;
+
+import static org.mockito.Mockito.*;
+
+public class AlterInstanceHandlerTest {
```
Comment (strongduanmu) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java:16]: @ismailalammar Please add final for AlterInstanceHandlerTest.

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (strongduanmu) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java:17]: Please remove this useless blank line.

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        ContextManager contextManager = mock(ContextManager.class, RETURNS_DEEP_STUBS);
+        MetaDataPersistService metaDataPersistService = new MetaDataPersistService(mock(PersistRepository.class));
+        when(contextManager.getMetaDataContexts().getMetaDataPersistService()).thenReturn(Optional.of(metaDataPersistService));
+        ProxyContext.getInstance().init(contextManager);
+        String instanceId = "instance_id";
+        String key = "xa_recovery_nodes";
+        String value = "value_1";
+        new AlterInstanceHandler().initStatement(getSQLStatement(instanceId , key , value)).execute();
+    }
+
```
Comment (strongduanmu) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java:49]: Please remove this useless blank line.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java (hunk 1 above)
Comment (ismailalammar) [reply shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java]: resolved

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java (hunk 2 above)
Comment (ismailalammar) [reply shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/AlterInstanceHandlerTest.java]: resolved

**Comment by ismailalammar** [comment]
HI @strongduanmu could you please help me to review the changes, CI pending for approval Thank you!

**Comment by strongduanmu** [comment]
> HI @strongduanmu could you please help me to review the changes, CI pending for approval Thank you!

@ismailalammar Thank you, I have approved ci task.

**Comment by ismailalammar** [comment]
> > HI @strongduanmu could you please help me to review the changes, CI pending for approval Thank you!
> 
> @ismailalammar Thank you, I have approved ci task.

Thanks @strongduanmu ,  i just saw all checks has been completed successfully!

**Review by strongduanmu — APPROVED** [review]
Thank you @ismailalammar, merged.

**Pull request merged**